docs/misc/changelog.rst

```diff
@@ -37,6 +37,7 @@ Bug Fixes:
 - Fixed ``VecExtractDictObs`` does not handle terminal observation (@WeberSamuel)
 - Set NumPy version to ``>=1.20`` due to use of ``numpy.typing`` (@troiganto)
 - Fixed loading DQN changes ``target_update_interval`` (@tobirohrer)
+- Fixed logger for Google Colab
```

Comment:
@araffin is it worth mentioning since the bug has been introduced by 2.0.0a7?

Reply:
no, but I would rather mention a breaking change in the way the logger check the file

Reply:
It would also be nice to add a regression test if possible

Reply:
How would you do that? By using the `OutStream` of `ipykernel` for a test?

Reply:
not sure, in that case I would create a dummy subclass of text io base and override stdout (that's what ipykernel does if get it)?
(best would be with ipykernel indeed, but I don't think it's part of SB3 dependencies)

Reply:
> I don't think it's part of SB3 dependencies

Agree

> create a dummy subclass of text io base

check https://github.com/DLR-RM/stable-baselines3/pull/1484/commits/45db2c14b71b7cfc09ae21b69805520d8443ac50